lib/mongoid/query_cache.rb

```diff
@@ -20,11 +14,7 @@ class << self
       # @return [ Hash ] The hash of cached queries.
       # @api private
       def cache_table
```

Comment:
If this method is not used by Mongoid I think we should get rid of it completely.

Reply:
done